toonz/sources/toonzqt/fxschematicnode.cpp

```diff
@@ -2525,12 +2364,48 @@ FxSchematicNormalFxNode::FxSchematicNormalFxNode(FxSchematicScene *scene,
   else if (isMatteFx(id))
     m_type = eNormalMatteFx;
 
+  switch (m_type) {
+  case eNormalFx:
+  case eMacroFx:
+  case eNormalLayerBlendingFx:
+  case eNormalMatteFx:
+  case eNormalImageAdjustFx: {
+    QString fxId = QString::fromStdWString(getFx()->getFxId());
+    if (m_name != fxId)
+      setToolTip(QString("%1 : %2").arg(m_name, fxId));
+    else
+      setToolTip(m_name);
+  } break;
+  case eZeraryFx: {
+    TZeraryColumnFx *zfx = dynamic_cast<TZeraryColumnFx *>(getFx());
+    if (zfx) {
+      TFx *zeraryFx = zfx->getZeraryFx();
+      if (zeraryFx) {
+        setToolTip(QString("%1 : %2").arg(
+            m_name, QString::fromStdWString(zeraryFx->getFxId())));
+      }
+    }
+    break;
+  }
+  case eGroupedFx: {
+    QString fxId =
+        "Group " + QString::number(getFx()->getAttributes()->getGroupId());
+    if (m_name != fxId)
+      setToolTip(QString("%1 (%2)").arg(m_name, fxId));
+    else
+      setToolTip(m_name);
+  }
+  }
+
   m_nameItem = new SchematicName(this, 72, 20);  // for rename
   m_outDock  = new FxSchematicDock(this, "", 0, eFxOutputPort);
   m_linkDock = new FxSchematicDock(this, "", 0, eFxLinkPort);
 
-  m_renderToggle = new SchematicToggle(
-      this, QPixmap(":Resources/schematic_prev_eye.png"), 0);
+  m_renderToggle =
+      new SchematicToggle(this, viewer->getSchematicPreviewButtonOnImage(),
+                          viewer->getSchematicPreviewButtonBgOnColor(),
+                          viewer->getSchematicPreviewButtonOffImage(),
+                          viewer->getSchematicPreviewButtonBgOffColor(), 0);
```

Comment:
`m_isLargeScaled`  seems to be missing from the arguments of `SchameticToggle` ctor.